src/sessions/softphone-session-handler.ts

```diff
@@ -523,14 +523,6 @@ export default class SoftphoneSessionHandler extends BaseSessionHandler {
   }
 
   async rejectPendingSession (pendingSession: IPendingSession): Promise<any> {
-    if (!this.hasActiveSession()) {
-      return super.rejectPendingSession(pendingSession);
-    }
```

Comment:
Can we send a PATCH even if we don't have a session?

Reply:
yes.